docs/learn-kusama-vs-polkadot.md

```diff
@@ -0,0 +1,57 @@
+---
+id: learn-kusama-vs-polkadot
+title: Comparing Polkadot and Kusama
+sidebar_label: Comparing Polkadot and Kusama
+---
+
+# What is the difference between Polkadot and Kusama?
+
+Although they are built on nearly the same codebase, Polkadot and Kusama are independent, standalone networks with different priorities.  
```

Comment:
Technically they are built from the _same_ codebase so it may be better to rephrase this as saying they "share many parts of their code".

Reply:
Changed